src/Node/Expression/CallExpression.php

```diff
@@ -297,17 +297,15 @@ private function reflectCallable($callable)
         }
         $r = new \ReflectionFunction($closure);
 
-        if (false !== strpos($r->name, '{closure}')) {
+        if (str_contains($r->name, '{closure}')) {
             return $this->reflector = [$r, $callable, 'Closure'];
         }
 
         if ($object = $r->getClosureThis()) {
             $callable = [$object, $r->name];
-            $callableName = (\function_exists('get_debug_type') ? get_debug_type($object) : \get_class($object)).'::'.$r->name;
-        } elseif (\PHP_VERSION_ID >= 80111 && $class = $r->getClosureCalledClass()) {
+            $callableName = (\function_exists('get_debug_type') ? get_debug_type($object) : $object::class).'::'.$r->name;
```

Comment:
`get_debug_type` will always exist.

```suggestion
            $callableName = get_debug_type($object).'::'.$r->name;
```

`get_debug_type` is used since https://github.com/twigphp/Twig/pull/3702 to support named closures.